libs/ledger-live-common/src/apps/filtering.ts

```diff
@@ -46,31 +44,24 @@ function typeFilter(
   updateAwareInstalledApps: UpdateAwareInstalledApps,
   installQueue: string[] = [],
 ) {
-  return ({ currencyId, name }: App) =>
-    filters.every(filter => {
-      const key = camelCase(`currency_${currencyId}`) as FeatureId;
-      const { enabled: currencyEnabled = true } = getFeature({ key }) ?? {};
```

Comment:
What was the reason to lift this logic outside the components ?

This makes it harder to use correctly as the parent component now needs to take care of that and not forget to update all places if there is an update to this logic

Reply:
This is a feature flag relative determination, so I prefer putting it in the front component where feature flag is potentially also needed for component rendering.

Reply:
Ok we should probably share the logic to filter Apps to avoid the duplication but lifting feature flags like that makes sense